styles.css

```diff
@@ -55,40 +55,40 @@
 .nav-ul {
```

Comment:
The mobile navbar is still not spanning 100% width. Please do so as requested by the previous reviewer.
Here is a snapshot of your app.
![image](https://user-images.githubusercontent.com/33268927/175055051-7f0df3fa-f311-4e63-b96d-7f5b5c3669e9.png)